app.conf.template

```diff
@@ -43,6 +43,11 @@ TRANSFORMER_AUTOSCALE_ENABLED = True
 
 TRANSFORMER_MANAGER_MODE = 'external-kubernetes'
 
+# Should we validate the docker image exists on DockerHub?
+# Set to False if you are doing local development and don't want to
+# push your image
+TRANSFORMER_VALIDATE_DOCKER_IMAGE = True
```

Comment:
Should this setting last for the entire lifetime of the ServiceX deployment, or should it be specified on a per-request basis?

Reply:
For 1.0 we are trying to keep the requests fairly simple and not yet a Swiss army knife. I think it will be rare for people to turn this off. Really just during development before you push the image to the repo. If you are running the system on a multi-node Kubernetes cluster you really have to have the image in a repository